pkg/config/samples/default.dusk.toml

```diff
@@ -206,6 +206,9 @@ accumulatorWorkers = 4
 
 # Information for the node to send consensus transactions with
 [consensus]
+# Path to a file that stores Consensus Keys / BLS public and secret keys
+# if file does not exist, it will be created at startup.
+keysfile = "/path/consensus.keys"
```

Comment:
The default path with `/path/` in it looks weird, but I could be missing something here...